<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Timer</class>
 <widget class="QMainWindow" name="Timer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>召唤师技能计时器</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLCDNumber" name="min">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="start">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stop">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>0</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>重置</string>
    </property>
   </widget>
   <widget class="QLabel" name="top">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>上单</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>闪现</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>闪现</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn7">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>闪现</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn10">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>闪现</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn13">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>闪现</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn8">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>点燃</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn5">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>疾跑</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn11">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>治疗</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn2">
    <property name="geometry">
     <rect>
      <x>139</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>点燃</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn14">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>点燃</string>
    </property>
   </widget>
   <widget class="QLabel" name="top_txt">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>80</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="jug_txt">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>110</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="mid_txt">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>140</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="adc_txt">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>170</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="sup_txt">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>200</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn9">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>传送</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn12">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>净化</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>传送</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn15">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>虚弱</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn6">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>点燃</string>
    </property>
   </widget>
   <widget class="QLabel" name="top_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>打野</string>
    </property>
   </widget>
   <widget class="QLabel" name="top_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>中单</string>
    </property>
   </widget>
   <widget class="QLabel" name="top_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ADC</string>
    </property>
   </widget>
   <widget class="QLabel" name="top_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>辅助</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
